spec/models/maths_and_physics/dqt_hecos_code_record_check_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe MathsAndPhysics::DQTRecord do
+  ELIGIBLE_MATHS_HECOS_CODES = %w[100400 100401 100402 100403 100404 100405 100406 101027 101028 101029 101030 101031 101032 101033 101034].freeze
+  ELIGIBLE_PHYSICS_HECOS_CODES = %w[100416 100419 100425 100426 101060 101061 101068 101071 101074 101075 101076 101077 101223 101300 101390 101391].freeze
+  EXAMPLE_ELIGIBLE_HECOS_CODES = ELIGIBLE_MATHS_HECOS_CODES + ELIGIBLE_PHYSICS_HECOS_CODES
+
+  EXAMPLE_NON_ELIGIBLE_HECOS_CODES = %w[100430 101066 101078 100300 100396 100427].freeze
+
+  describe "#eligible hecos code" do
```

Comment:
Usually in unit tests you `describe` the method under test. The contexts and `it` blocks will describe the condition or behaviour you're testing.
```suggestion
  describe "#eligible?" do
```